src/utils/helpers.ts

```diff
@@ -68,21 +68,21 @@ export const isValidDate = (d: string) => {
  * @param experience - an attorney's required experience for a listed case.
  * @returns appropriate string description according to the experience level.
  */
-export const parseExperience = (experience: ImmigrationLawExperienceEnum) => {
-  if (experience === 'LOW') {
-    return `${
-      experience.charAt(0).toUpperCase() + experience.toLowerCase().substring(1)
-    } (No prior case)`;
-  }
-  if (experience === 'MEDIUM') {
-    return `${
-      experience.charAt(0).toUpperCase() + experience.toLowerCase().substring(1)
-    } (One prior case)`;
-  }
-  return `${
-    experience.charAt(0).toUpperCase() + experience.toLowerCase().substring(1)
-  } (Multiple prior cases)`;
-};
+// export const parseExperience = (experience: ImmigrationLawExperienceEnum) => {
```

Comment:
since we don't need `immigration_law_experience` on the `Profile` schema, can we just delete these commented out lines of code (and anywhere else it was imported `CaseDetails` component i think) + remove the field from the schema

Reply:
sg!

Reply:
lemme check if i commented it out for the profile schema